src/scene/materials/standard-material.js

```diff
@@ -72,8 +72,8 @@ import { Quat } from '../../math/quat.js';
  * @property {boolean} specularVertexColor Use mesh vertex colors for specular. If specularMap or are specularTint are set, they'll be multiplied by vertex colors.
  * @property {string} specularVertexColorChannel Vertex color channels to use for specular. Can be "r", "g", "b", "a", "rgb" or any swizzled combination.
  *
- * @property {boolean} enableGGXSpecular Enables GGX specular. Also enables anisotropy parameter to set material anisotropy.
- * @property {number} anisotropy Defines amount of anisotropy. Requires enableGGXSpecular is set to true.
+ * @property {boolean} enableGGXSpecular Enables GGX specular. Also enables {@link pc.Material#anisotropy}  parameter to set material anisotropy.
+ * @property {number} anisotropy Defines amount of anisotropy. Requires {@link pc.Material#enableGGXSpecular} is set to true.
```

Comment:
Yes - also fixed some other links I added before. I added these links because I was referencing the API docs for new tooltips in the editor and the API docs are sorted differently from in the code which can make interrelated properties more difficult to figure out.